source/develop/release-management/features/storage/qcow2v3.html.md

```diff
@@ -228,6 +228,14 @@ Here is an example of the reponse output:
 
 These operations will be disabled if the VM is running or if those disks are locked (for example copy, move, delete).
 
+#### General Information
+1. The "Default" Template's QCOW volume will have an arbitrary QCOW compat version of 0.10 compat version.
```

Comment:
Because it is marked in the DB as QCOW volume, (although it is just an transient volume).
Since QCOW volumes are always set with 0.10 or 1.1 I've updated it to 0.10 since it isn't a QCOW volume anyhow...